src/Resources/config/validation/GiftCardConfiguration.xml

```diff
@@ -9,6 +9,13 @@
             <option name="fields">code</option>
             <option name="groups">setono_sylius_gift_card</option>
         </constraint>
+        <constraint name="Setono\SyliusGiftCardPlugin\Validator\Constraints\HasBackgroundImage">
+            <option name="groups">
+                <value>setono_sylius_gift_card</value>
+                <value>setono_sylius_gift_card_create</value>
```

Comment:
Why do we need the `setono_sylius_gift_card_create` here?

Reply:
We don't anymore